GPU/GPUCommon.cpp

```diff
@@ -973,7 +973,7 @@ void GPUCommon::FastRunLoop(DisplayList &list) {
 	int dc = downcount;
 	for (; dc > 0; --dc) {
 		// We know that display list PCs have the upper nibble == 0 - no need to mask the pointer
-		const u32 op = *(const u32 *)(Memory::base + list.pc);
+		const u32 op = Memory::ReadUnchecked_U32(list.pc);
```

Comment:
nit: Probably fine, but the reason for this before was to avoid masking the pointer.

Maybe it's not worth it anymore (this was done when 32-bit was more common), but we should remove the comment above if we're not doing that anymore.

-[Unknown]

Reply:
ah I totally missed that comment, I only changed it to `Memory::ReadUnchecked_U32` to match the rest of the code, ill change that back then and cast to `u32_le*` instead.
WiiU is 32-bit, so it would make sense to keep it the way it was.